### ISSUE TITLE
[question] astral support

### ISSUE THREAD

**Issue by timosachsenberg** [compomics/ThermoRawFileParser#167]
Hi,
did anyone successfully convert astral data with ThermoRawFileParser?
Any experience to share?

**Comment by caetera** [comment]
Hi @timosachsenberg ,

it should be possible already now with `-e, --ignoreInstrumentErrors` enabled. I could not (so far) grab some data from "real" Astral, i.e. with the release software. The files I saw so far use pre-release software; thus, I don't know if there are any problems. If you can share some examples I am happy to take a look.

**Comment by timosachsenberg** [comment]
I have some data and I will try to figure out if I can share them.
Is there an easy way to find out if it was recorded with the pre-release or release software?

**Comment by caetera** [comment]
Running TRFP to extract only metadata as TXT on one of the raw files deposited in PRIDE, i.e. `thermorawfileparser -m 1 -f none -i {astral_rawfile}` I get
```
Instrument name=OLEP08
...
Software version=[NCIT, NCIT:C111093, Software Version, 0.7-0.7.114.0/0.7.53.0]
```
To the best of my knowledge, the release version should be `1.xx` and the instrument name should probably mention __Astral__

**Comment by timosachsenberg** [comment]
Hmm then I probably also have only pre-release data
```
RAW file version=66
Creation date=04/28/2023 19:03:34
#InstrumentProperties
Instrument model=[MS, MS:1000494, Thermo Scientific instrument model, Orbitrap Astral]
Instrument name=OLEP03
Instrument serial number=[MS, MS:1000529, instrument serial number, Invalid_SN_0001]
Software version=[NCIT, NCIT:C111093, Software Version, 0.8-0.8.99.0-flavor=debug/0.8.70.0]
Firmware version=rev. 1
```

**Comment by timosachsenberg** [comment]
BTW I converted the3gb raw file to a 2.5gb mzML file (???).
The OpenMS FileInfo tool I get all precursor annotated as charge 0.
```
MS levels: 1, 2
Total number of peaks: 207343040
Number of spectra: 46974

Ranges:
  retention time: 0.00 .. 2388.02 sec (39.8 min)
  mass-to-charge: 0.00 .. 1600.00
  intensity:      18.40 .. 2683463424.00

Number of spectra per MS level:
  level 1: 5498
  level 2: 41476

Peak type from metadata (or estimated from data)
  level 1: Centroid (Centroid)
  level 2: Centroid (Centroid)

Activation methods
    MS-Level 2 & HCD (beam-type collision-induced dissociation): 41476

Precursor charge distribution:
  charge 0: 41476x
```

so there might be something off during conversion.

**Comment by caetera** [comment]
The size is indeed quite big for Astral scans, they have a few thousand peaks often (especially for DIA MS2), so that part won't change a lot.
I will need to look into precursor charge issue, maybe there's something new, however, in case of DIA charge for MS2 scans is often unassigned.

If you don't mind, you can build the latest beta of 1.4.3 from the the corresponding branch in GitHub. It has some early support for Astral.

**Comment by timosachsenberg** [comment]
I got access to two test files. Can you contact me, so I can share them privately?

**Comment by caetera** [comment]
Released in version 1.4.3
